src/services/build-service.ts

```diff
@@ -31,8 +31,12 @@ export default class BuildService extends PercyClientService {
   async finalize() {
     if (!this.buildId) { return }
 
-    await this.percyClient.finalizeBuild(this.buildId).catch(logError)
-    this.logEvent('finalized')
+    try {
+      await this.percyClient.finalizeBuild(this.buildId)
+      this.logEvent('finalized')
+    } catch (err) {
+      logError(err)
```

Comment:
Can we add some context here..  Perhaps a log line above this logError that says `Error during build finalize:` or something similar.